libraries/appbase/include/appbase/application.hpp

```diff
@@ -88,6 +88,9 @@ namespace appbase {
 
          boost::asio::io_service& get_io_service() { return *io_serv; }
 
+         const std::string& get_chain_desc() const{ return chain_desc; };
```

Comment:
No, nothing about the blockchain should be in appbase. Appbase is application agnostic.